src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -847,7 +816,7 @@ BtreeLBAManager::_update_mapping(
   LogicalChildNode* nextent)
 {
   auto c = get_context(t);
-  return with_btree_ret<LBABtree, update_mapping_ret_bare_t>(
+  return with_btree<LBABtree>(
```

Comment:
Same here.

Reply:
Seems they were not written on the same day... `with_btree` works well for two cases.

Reply:
Can we stick to the simpler `with_btree` then ?

CC @athanatos